toolkits/springernature/packages/springernature-user-details/scss/50-components/enhanced.scss

```diff
@@ -25,15 +25,14 @@
 
 	&__content-item {
 		padding: 0 12px;
```

Comment:
🤔  Sorry I know you didn't add this in as part of these changes but I'm wondering if this padding should actually also be placed in core.scss like the font size is.

Reply:
No that's not required for the core version.